handle-multipart-upload.go

```diff
@@ -122,6 +123,7 @@ func HandleMultipartRequest(f *proxy.Flow) error {
 			return fmt.Errorf("error reading  multipart request: %v", err)
 		}
 
+		latencyStart := time.Now()
```

Comment:
it's better to move the logic of capturing elapsed time to [encrypt-decrypt.go](https://github.com/byronwhitlock-google/go-gcsproxy/blob/main/encrypt-decrypt.go)

Reply:
yes I initially added the code there but later moved it here since we are also using methods like [checkKms()](https://github.com/byronwhitlock-google/go-gcsproxy/blob/2326b51ad09d69406d698e6b00c12ab7937ee2a7/main.go#L150), which will also call the encrypt method but we do not want to capture its latency.
Let me know if it still makes sense to move it to encrypt-decrypt.go, and I will move it there.

Reply:
checkKms is more like checkEncryption. It's only called once. I don't think it matters that much for performance testing. I'd still put all the latency capture code in encrypt/decrypt functions. If you really want to exclude checkKms, you can probably leverage the ctx param in the encrypt/decrypt functions. i.e checkKms can set the ctx.latencyMetricDisabled=true.